docs/appendices/release-notes/unreleased.rst

```diff
@@ -141,4 +144,7 @@ Fixes
 
 - Fixed an issue that caused showing an incorrect log message in case of an
   authentication failure. "Password authentication" used to be shown instead
-  of the actual authentication method name.
\ No newline at end of file
+  of the actual authentication method name.
+
+- Fixed an issue in the administration console: Nested arrays in
```

Comment:
We shouldn't have fixes within a feature release.

Would it be possible to do a 1.19.2 with the fix - backport that to CrateDB 4.6, and remove the entry from this PR?

Reply:
Thanks. I just removed this entry from the PR and will hand in another PR only including this fix, for targeting the 4.6 series.

Reply:
See #11776.